<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,1" columnstretch="0,1,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="searchPushButton">
        <property name="text">
         <string>목록조회</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>경로</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="lrcCreatorPushButton">
        <property name="text">
         <string>LRC생성</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
      <item row="1" column="0" colspan="5">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>LRC</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>파일명</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>자료형</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>앨범</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>타이틀</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>가수</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="directoryPushButton">
        <property name="text">
         <string>경로검색</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
